packages/babel-helper-member-expression-to-functions/src/index.js

```diff
@@ -329,6 +329,7 @@ const handle = {
       return;
     }
 
+    // for ( MEMBER of ARR) && for ( MEMBER in ARR)
```

Comment:
```suggestion
    // for (MEMBER of ARR)
    // for (MEMBER in ARR)
```
nit: `&&` could cause confusion since it is a valid JavaScript operator.